### ISSUE TITLE
Budget Calendar

### ISSUE THREAD

**Issue by datapike** [excid3/simple_calendar#126]
I'm trying to repurpose this gem to make a budget calendar. Similar to PNC Virtual Wallet, but one that prints on each day the events(transactions) and the forecasted account balance. I have the transactions working and printing onto the calendar, but I can't get the account balance to print, nor can I figure out how to get it to dynamically update that balance based on the transactions on that day. I'm also trying to make the transactions recurring, but that's escaping me too.

Basically I want to be able to either click on a day and it shows me the forecasted balance in a sidebar, OR just show the forecasted balance on every day of the calendar, which I thought could be easier...

Any and all help would be greatly appreciated. Thanks.

**Comment by excid3** [comment]
I'm not familiar with PNC Virtual Wallet, but it sounds like you might want to list the transactions on each day of the calendar and then use some JS to listen to when the user clicks on a date. You can use data attributes to set like data-total on each transaction and then have the Javascript add these up to display somewhere. That's probably what I would do based on what you've described so far. That make sense?

**Comment by datapike** [comment]
I haven't really used JS with Rails much yet, so I'm not sure how to do that. But even if I could just get the running balance to show on each day, that would be awesome. No clicking or JS required I think then. 

But I can't seem to get the showing of the account balance to work inside the month_calendar loop. I have the transactions linked to the account model and the account owns all the transactions. So there's an account_id on transactions which I'm trying to use to pull the account.balance. But it's not working. Past that, I'm not sure how the balance and transactions would then add themselves together.

**Comment by datapike** [comment]
I'm also having trouble getting the styling to hit the Month Year part between "previous" and "next". I can't get just the day number to show on each day either, it shows the whole date instead. Basically, I want it to look more like the example screenshot, and I can't figure out how to do that...

**Comment by excid3** [comment]
I'm not sure I added any classes to the header links, but if you've added the views to your Rails app to customize them, you can add some in there and target them with your own CSS pretty easily. That should fix that problem for you.

**Comment by datapike** [comment]
Thanks. I put the line in a span and added a class. That was simpler than expected. And now I feel dumb. Heh. 

The final thing I'm trying to do with this app is set up recurring transactions. I saw another post about it, but I haven't delved into it yet. Thanks for all your help.
